xml/security-security_features.xml

```diff
@@ -6,15 +6,43 @@
  <!ENTITY % entities SYSTEM "entity-decl.ent"> %entities;
 ]>
 <chapter xmlns="http://docbook.org/ns/docbook" xmlns:xi="http://www.w3.org/2001/XInclude" xmlns:xlink="http://www.w3.org/1999/xlink" xml:id="newsecurity">
- <title>&productnamereg;: Security Features Overview</title>
+ <title>&productnamereg;: Security Planning and Features</title>
+ <section xml:id="idg-all-security-security_planning-xml-1">
+  <title>Security Planning</title>
+  <para>
+   &productnamereg; is a complex system which has many options which
+   affect the security of the system and the workloads running within it. Careful planning
+   is required to ensure the correct options and features are chosen for your particular
+   environment. You should read
+   <link xlink:href="https://docs.openstack.org/security-guide/introduction/security-boundaries-and-threats.html">Security Boundaries and Threats</link>
+   to understand how OpenStack defines security domains. Ensure that the network layout
+   you choose for your deployment provides adequate network separation for your specific
+   requirements. The other parts of the
+   <link xlink:href="https://docs.openstack.org/security-guide/">OpenStack Security Guide</link>
+   are also informative.
+   Take advantage of the security features described below in this section. 
+  </para>
+  <para>
+   OpenStack service processes need to run some commands as root in order to provide
+   functionality. The service processes run as non-root users but can escalate privilege
+   to root to perform certain operations. If a service process is compromised to the
+   extent that an attacker can execute arbitrary commands as the service user then it is
+   likely that the attacker can elevate privilege to root and take over the system on
+   which the service is running. When planning the security controls for an OpenStack
+   deployment you should consider the services to be running as root when deciding which
+   security controls are appropriate for your environment. Timely installation of
+   security patches is essential. Other controls such as Intrusion Prevention Systems
+   and Web Application Firewalls should be considered. 
+  </para>
+ </section>
  <section xml:id="idg-all-security-security_features-xml-6">
-  <title>Security features in &product;</title>
+  <title>Security Features in &product;</title>
   <para>
    Enterprises need protection against security breaches, insider threats, and
    operational issues that increase the risk to sensitive data. &product;
    provides capabilities that help you
    protect your data at rest and in transit, enable centralized key management,
-   and comply with the Payment Card Industry Data Security Standard (PCI-DSS).
+   and meet compliace requirements.
```

Comment:
s/compliace/compliance/

Reply:
fixed in https://github.com/SUSE-Cloud/doc-cloud/pull/950